docs/guide/db-dao.md

```diff
@@ -638,7 +638,7 @@ $rows = Yii::$app->db->useMaster(function ($db) {
 You may also directly set `Yii::$app->db->enableSlaves` to be `false` to direct all queries to the master connection.
 
 
-## Working with Database Schema <span id="database-schema"></span>
```

Comment:
That would change the anchor so all the existing links will be broken because of the change. That's probably OK but...

Reply:
Why? It is not working very well in the current offline version. Open my [forked repo](https://github.com/toir427/yii2/blob/documentation/docs/guide/tutorial-yii-integration.md#using-both-yii2-with-yii) and check that line. It is worked very well offline and online. And there is also an anchor.

Reply:
`<span>` tag disrupting for find link
![Uploading Screenshot-20201101115402-694x121.png…]()

Reply:
OK. Nothing to break then :)